commit 6b3f1d3c1a278036da4d650c7fedc1b1a537b5d1
Author: Verevka <verevka86@gmail.com>
Date:   Tue Jun 27 11:44:38 2017 +0300

    Added IFNAMSIZ
    
    Change-Id: I833f28dd73efd2dc053f0dea625d963719a0fa96

diff --git a/include/ebtables_u.h b/include/ebtables_u.h
index 63ce231..2655a67 100644
--- a/include/ebtables_u.h
+++ b/include/ebtables_u.h
@@ -21,6 +21,8 @@
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#define IFNAMSIZ		16
+
 #ifndef EBTABLES_U_H
 #define EBTABLES_U_H
 #include <netinet/in.h>
