From c76d952cfae850739c7fd9258269b5819c36bee5 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 01 Sep 2017 21:35:03 -0700
Subject: [PATCH] fingerprint: Don't fail on fingerprint2.0

Change-Id: I522d0fc000e7a0190ab8680ed3d3602e4bc459ea
---

diff --git a/biometrics/fingerprint/2.1/default/BiometricsFingerprint.cpp b/biometrics/fingerprint/2.1/default/BiometricsFingerprint.cpp
index c6774ca..04d1ae7 100644
--- a/biometrics/fingerprint/2.1/default/BiometricsFingerprint.cpp
+++ b/biometrics/fingerprint/2.1/default/BiometricsFingerprint.cpp
@@ -33,6 +33,7 @@
 namespace implementation {
 
 // Supported fingerprint HAL version
+static const uint16_t kVersionOld = HARDWARE_MODULE_API_VERSION(2, 0);
 static const uint16_t kVersion = HARDWARE_MODULE_API_VERSION(2, 1);
 
 using RequestStatus =
@@ -241,7 +242,10 @@
     if (kVersion != device->version) {
         // enforce version on new devices because of HIDL@2.1 translation layer
         ALOGE("Wrong fp version. Expected %d, got %d", kVersion, device->version);
-        return nullptr;
+        // don't fail on HAL version 2.0, it works just fine
+        if (device->version < kVersionOld) {
+            return nullptr;
+        }
     }
 
     fingerprint_device_t* fp_device =
